README.md

```diff
@@ -24,7 +24,7 @@ Currently, Zettle provides APIs for the following markets:
 ## APIs
 
 -   [OAuth](authorization.adoc)
--   [Finance](finance.adoc)
+-   [Finance](finance-api/overview.md)
```

Comment:
This overview is not present in pusher docs which makes it inconsistent with templates and existing docs.

Reply:
You're right. :) The plan is to iterate the templates and update the Pusher API after the Finance API doc changes.